enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -215,13 +233,34 @@
     (.mkdirs f)
     (when-not (.canWrite f)
       (throw (ex-info (format "Destination path is not writeable: %s" path) {:filename path}))))
-  (serdes/with-cache
-    (-> (cond-> opts
-          (seq collection-ids) (assoc :targets (v2.extract/make-targets-of-type "Collection" collection-ids)))
-        v2.extract/extract
-        (v2.storage/store! path)))
-  (log/info (trs "Export to {0} complete!" path) (u/emoji "🚛💨 📦"))
-  ::v2-dump-complete)
+  (let [start  (System/currentTimeMillis)
+        err    (atom nil)
+        report (try
+                 (serdes/with-cache
+                   (-> (cond-> opts
+                         (seq collection-ids)
+                         (assoc :targets (v2.extract/make-targets-of-type "Collection" collection-ids)))
+                       v2.extract/extract
+                       (v2.storage/store! path)))
+                 (catch Exception e
+                   (reset! err e)))]
+    (snowplow/track-event! ::snowplow/serialization-export nil
+                           {:source          "api"
```

Comment:
Aside: it would be nice if `track-event!` took care of turning kebab-case into snake_case. Underscores hurt my eyes in Clojure 😄